migrations/20231129172339_job_queue_table.sql

```diff
@@ -9,15 +9,20 @@ CREATE TABLE job_queue(
     id BIGSERIAL PRIMARY KEY,
     attempt INT NOT NULL DEFAULT 0,
     attempted_at TIMESTAMPTZ DEFAULT NULL,
-    attempted_by TEXT[] DEFAULT ARRAY[]::TEXT[],
-    completed_at TIMESTAMPTZ DEFAULT NULL,
+    attempted_by TEXT [] DEFAULT ARRAY [] :: TEXT [],
     created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
-    errors jsonb[],
+    errors jsonb [],
     max_attempts INT NOT NULL DEFAULT 1,
     finished_at TIMESTAMPTZ DEFAULT NULL,
     parameters JSONB,
-    queue TEXT NOT NULL DEFAULT 'default'::text,
+    queue TEXT NOT NULL DEFAULT 'default' :: text,
     scheduled_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
-    status job_status NOT NULL DEFAULT 'available'::job_status,
+    status job_status NOT NULL DEFAULT 'available' :: job_status,
```

Comment:
Ew, I use the vscode `SQL Formatter` extension and disagree with its choices, one sec.